.idea/compiler.xml

```diff
@@ -0,0 +1,13 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Is this necessary? Can you please remove this from the tracked files if it is not necessary?

Reply:
This file came from gitignore, no big deal